docs/operators/rest.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Rest API on top of GraphQL
+description: Exposing REST endpoints on top of GraphQL via @rest directive.
+---
+
+Tailcall's `@rest` directive exposes standard REST endpoints, enabling the fetching of data from REST APIs directly through GraphQL queries.
+
+This guide will show you how to use the `@rest` directive to create a RESTful service over a GraphQL layer for retrieving product information.
+
+## The @rest Directive Basics
+
+- **method**: Specifies the HTTP method (GET, POST, etc.).
+
+- **path**: Defines the endpoint URL, supporting dynamic values from query arguments.
+
+## Example: Retrieving Product Information
```

Comment:
Let's use a posts example from Jsonplaceholder. The original API when `/posts` is hit returns posts with only userId, here we should join it with the users API. That way when we hit `/posts` from tailcall, we get posts with some basic user information than just userId.